bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: operators.coreos.com/v1alpha1
+apiVersion: operators.coreos.com/v1
```

Comment:
hum, i think we need rework on the "bundle" part.
not sure why this is updated to v1 on CSV